packages/sanity/src/studio/AuthBoundary.tsx

```diff
@@ -25,16 +29,27 @@ export function AuthBoundary({
 
   useEffect(() => {
     activeWorkspace.auth.state.subscribe({
-      next: ({authenticated}) => setLoggedIn(authenticated ? 'logged-in' : 'logged-out'),
+      next: ({authenticated, currentUser}) => {
```

Comment:
I realise this isn't introduced by this PR, but we need to unsubscribe in the effect cleanup:
```js
useEffect(() => {
  const subscription = activeWorkspace.auth.state.subscribe(/*…*/)
  return () => {
      subscription.unsubscribe()
    }
  }, [activeWorkspace])
```

Reply:
Thx 👍